transforms/language/html2parquet/ray/README.md

```diff
@@ -0,0 +1,17 @@
+# html2parquet Ray Transform 
+
+This module implements the ray version of the [html2parquet transform].
+
+The HTML conversion is using the [Trafilatura](https://trafilatura.readthedocs.io/en/latest/usage-python.html).
+
+## Prometheus metrics
+
+The transform will produce the following statsd metrics:
+
+| metric name                      | Description                                                      |
```

Comment:
These are not part of the output parquet file.  I think you need to tell the reader where these can be viewed.   Also, please link from here to the python readme for annotations and configuration.

Reply:
Actually, I'm not entirely sure where these metrics can be viewed. In the Ray implementation, I borrowed the pdf2parquet code, but I’m not very familiar with Prometheus metrics. These metrics are generated from the `Html2ParquetRayTransform` class, and they can be viewed through logging. After some investigation, it appears that you can access these metrics in the Prometheus dashboard under the relevant namespace for your application. However, my experience with this is limited. I would greatly appreciate your assistance in adding more detailed descriptions!